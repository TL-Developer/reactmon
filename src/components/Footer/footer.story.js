'use strict';

import React from 'react';
import { storiesOf } from '@storybook/react';
import { action } from '@storybook/addon-actions';
import { linkTo } from '@storybook/addon-links';
import Footer from './index'

const stories = storiesOf('Footer', module)

stories.add('first story', () => (
  <Footer />
))
