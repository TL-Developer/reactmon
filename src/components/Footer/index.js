import React from 'react'
import './style.scss';

const Footer = () => (
  <div className="CcFooter">
    Footer
  </div>
);

export default Footer;
