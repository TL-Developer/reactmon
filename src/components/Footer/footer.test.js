'use strict';

import React from 'react';
import renderer from 'react-test-renderer';
import Footer from './index';

it('Should Footer to match snapshot', () => {
  const tree = renderer.create(<Footer />).toJSON();
  expect(tree).toMatchSnapshot();
});
