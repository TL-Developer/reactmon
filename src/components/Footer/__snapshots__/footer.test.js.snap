// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should Footer to match snapshot 1`] = `
<div
  className="CcFooter"
>
  Footer
</div>
`;
